java/client/src/main/java/redis/clients/jedis/compatibility-layer-migration-guide.md

```diff
@@ -0,0 +1,246 @@
+# Valkey GLIDE Compatibility Layer Migration Guide
+
+## Overview
+
+The Valkey GLIDE compatibility layer enables seamless migration from Jedis to Valkey GLIDE with minimal code changes. This guide covers supported features, migration steps, and current limitations.
+
+## Quick Migration
+
+### Step 1: Update Dependencies
```

Comment:
Do we want to add an "Advanced Migration" section mentioning how to switch without a recompile? That would be changing the application's classpath such that it does not have the Redis Jedis JAR and instead has Glide + the Jedis compatibility layer?